### ISSUE TITLE
Logging issues at startup in openhab.log

### ISSUE THREAD

**Issue by robnielsen** [openhab/openhab-core#1733]
I see the following in openhab.log:

```
2020-10-14 16:59:40.497 [ERROR] [org.openhab.core.items.GenericItem  ] - Tried to set invalid state 0.0 (DecimalType) on item bathFans of type SwitchItem, ignoring it
```

The .items file is similar to:

```
Group:Switch:OR(ON, OFF) bathFans    "All Bath Fans [%d are on]" (all)
Switch mMainBathFan   (bathFans) {channel=""}
Switch sMasterBathFan (bathFans) {channel=""}
Switch sBathFan       (bathFans) {channel=""}
Switch bBathFan       (bathFans) {channel=""}
```

**Comment by wborn** [comment]
Do you also use mapdb to restore values on startup? The version used with OH3 was rewritten and seems to have some issues.

**Comment by robnielsen** [comment]
No, I don't use mapdb at all, is it enabled by default?

**Comment by wborn** [comment]
No it's not enabled by default. But I have also seen similar error logging about invalid state. I hope to find the root cause. :-)

**Comment by robnielsen** [comment]
It does appear that state is somehow persisted across restarts

**Comment by wborn** [comment]
It could be that you are now using rrd4j to restore the state on startup. IIRC rrd4j doesn't store the unit with quantity type items so that might cause your other issue https://github.com/openhab/openhab-core/issues/1732.

**Comment by robnielsen** [comment]
Yes, it does appear that it is installed. I didn't configure any sort of persistence. It isn't enabled:
![image](https://user-images.githubusercontent.com/838221/96507666-1aeda480-121f-11eb-91b1-e8853589ccce.png)

**Comment by robnielsen** [comment]
I uninstalled rrd4j and the errors went away.

**Comment by wborn** [comment]
OK that at least explains some of the issues. When a persistence service is installed [it is enabled and it will use the default persistence strategies](https://github.com/openhab/openhab-core/blob/7d70a97b77c1d79ae63271703368adccb588cc46/bundles/org.openhab.core.persistence/src/main/java/org/openhab/core/persistence/internal/PersistenceManagerImpl.java#L107-L116) of the service if none are are configured, which seems to be [storing items every minute and restoring on startup](https://github.com/openhab/openhab-addons/blob/08ce65a939f593f3a820af9cbd4b1737262733fa/bundles/org.openhab.persistence.rrd4j/src/main/java/org/openhab/persistence/rrd4j/internal/RRD4jPersistenceService.java#L606).

The default service setting is only used to determine which source should be used for charts.

**Comment by robnielsen** [comment]
I didn't install a persistence service. I'm using the conf/services/addons.cfg, and persistence isn't even defined.

**Comment by wborn** [comment]
It's installed by default and that is also not obvious in the setup wizard, see https://github.com/openhab/openhab-webui/pull/306#issuecomment-678621976. So it doesn't help that its default persistence strategies will now also start generating these issues for everyone out of the box.

**Comment by kaikreuzer** [comment]
We'd have to investigate why this happens - for me, restoring switch items works flawlessly with rrd4j.
But I guess we can assume that the problem is within the rrd4j persistence service, so I suggest to move the issue to openhab-addons.
